src/views/bootablevolumes/list/components/BootableVolumesRow.tsx

```diff
@@ -29,19 +29,14 @@ const BootableVolumesRow: FC<
   return (
     <>
       <TableData id="name" activeColumnIDs={activeColumnIDs} className="pf-m-width-20">
-        <Split hasGutter>
-          <SplitItem>
-            <ResourceLink
-              groupVersionKind={getBootableVolumeGroupVersionKind(obj)}
-              name={obj?.metadata?.name}
-            />
-          </SplitItem>
-          {obj.kind === DataSourceModel.kind && isDataSourceCloning(obj as V1beta1DataSource) && (
-            <SplitItem>
-              <Label>{t('Clone in progress')}</Label>
-            </SplitItem>
-          )}
-        </Split>
+        <ResourceLink
+          groupVersionKind={getBootableVolumeGroupVersionKind(obj)}
+          name={obj?.metadata?.name}
+          inline
```

Comment:
can you also add namespace prop to here? if you click on any pvc it moves to it's page and getting a 404 error for a second

Reply:
Ok. PTAL: https://github.com/kubevirt-ui/kubevirt-plugin/pull/1324